_data/connect/common/database-sources.yml

```diff
@@ -125,7 +138,7 @@ binlog-replication:
     description: |
       If `true`, Log-based Incremental Replication will be used as the default Replication Method for tables set to replicate. This choice can be overriden on individual tables at any time.
```

Comment:
Want to make note of this, as far as connect is concerned, does the tap need to do anything to respect this? or does the API fill it in on selection. I know the UI will fill it in, but not sure about Connect.